### ISSUE TITLE
pylint complains about the `processes` var initialization in `server.serve_multiple` file. 

### ISSUE THREAD

**Issue by ketan86** [huge-success/sanic#1564]
This may not affect the functionality but good to fix to make code more readable, understandable in terms of flow and also to take care of pylint error. 

**Pylint Error** 
```json
{
	"resource": "/Users/kpatel/PycharmProjects/personal/sanic/sanic/server.py",
	"owner": "_generated_diagnostic_collection_name_#1",
	"code": "undefined-variable",
	"severity": 4,
	"message": "Undefined variable: 'processes'",
	"source": "Python",
	"startLineNumber": 847,
	"startColumn": 24,
	"endLineNumber": 847,
	"endColumn": 33
}
```

**Code snippet**
```python
    def sig_handler(signal, frame):
        logger.info("Received signal %s. Shutting down.", Signals(signal).name)
        for process in processes:
            os.kill(process.pid, SIGTERM)

    signal_func(SIGINT, lambda s, f: sig_handler(s, f))
    signal_func(SIGTERM, lambda s, f: sig_handler(s, f))

    processes = []

    for _ in range(workers):
        process = Process(target=serve, kwargs=server_settings)
        process.daemon = True
        process.start()
        processes.append(process)

    for process in processes:
        process.join()

```

**Screen Shot** 
<img width="794" alt="Screen Shot 2019-05-05 at 10 13 30 AM" src="https://user-images.githubusercontent.com/5973097/57197691-42fc2180-6f1f-11e9-803d-5d8cbc19606a.png">

**Comment by ketan86** [comment]
Minor fix that i can take care if required.

**Comment by ahopkins** [comment]
@ketan86 Have at it! 🤘

**Comment by ahopkins** [comment]
Closing because implemented in #1565. Thanks @ketan86
